cloudsim-plus-testbeds/src/main/java/org/cloudsimplus/sla/readJsonFile/instancesConfigurationsJsonFile/PricingReader.java

```diff
@@ -0,0 +1,46 @@
+package org.cloudsimplus.sla.readJsonFile.instancesConfigurationsJsonFile;
+
+import com.google.gson.Gson;
+import hostFaultInjection.HostFaultInjectionExperiment;
+import org.cloudbus.cloudsim.util.ResourceLoader;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+
+/**
+ * This class was created to read a json file where is
+ * the instance and configurations about this.
+ *
+ * Created by raysaoliveira on 01/06/17.
+ */
+public class PricingReader {
+
+    private final AwsInstancesConfigurations configurations;
```

Comment:
change to: private final AwsEC2Instance instance;
Change all references to the AwsInstancesConfigurations with AwsEC2Instance.